uiowa/uiowa
# [ ITS ] Service content type fields

Resolves #6806 





# How to test

Currently importing the full site split leads to an error, but removing `config/sites/its.uiowa.edu/schemadotorg.schemadotorg_mapping.node.service.yml` prior to import gets around this. After the site split has been imported, you can restore this and then import it by itself.



One liner:

```
rm config/sites/its.uiowa.edu/schemadotorg.schemadotorg_mapping.node.service.yml && ddev blt ds --site=its.uiowa.edu && ddev drush @its.local config:import --source ../config/sites/its.uiowa.edu --partial -y && ddev drush @its.local cim -y && git restore config/ && ddev drush @its.local cim -y && ddev drush @its.local cr && ddev drush @its.local uli /node/add/service
```

Add a service or two (or three--go wild!)
Check out the new fields, and try to recreate some services https://its.uiowa.edu/all-services-and-software (note: Support and Training and News sections will come later). See that the field-types and formats make sense and fit for the type of data needed
(Note: You'll need to create Audience and Service Category taxonomy terms for those fields to appear and function)
Compare the full node display with the finalized mockup: https://sandbox.prod.drupal.uiowa.edu/its-service-office-365-mockup
Check the functionality for the Fees and Fees info fields, and that it matches the description at https://github.com/uiowa/uiowa/pull/6821#issuecomment-1707241922

On a Page, add a Featured Content block in Layout Builder
Include a service (or services)
Check the teaser functions with title summary (or trimmed body if no summary)
